sdk/dslink/src/main/java/org/dsa/iot/dslink/methods/responses/InvokeResponse.java

```diff
@@ -85,6 +87,23 @@ public void populate(JsonObject in) {
             results = new Table();
         }
         {
+        	JsonObject metaData = in.get("meta");
```

Comment:
not sure I like those scoped blocks. If you're just using this to avoid name collisions in variables, you can just extract a private method.